portal-impl/src/portal.properties

```diff
@@ -6389,6 +6389,11 @@
     #
     feature.flag.LPS-153332=false
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER5__NUMBER3__NUMBER4__NUMBER7__NUMBER8_
+    #
+    feature.flag.LPS-153478=false
```

Comment:
This FF does not belong to your PR. Please, remove it from the PR

Reply:
Ok, I'm on it